packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -109,6 +117,69 @@ export interface PersistedQueryManifest {
   operations: PersistedQueryManifestOperation[];
 }
 
+/**
+ * Source documents from a persisted documents manifest generated by [GraphQL
+ * Codegen](https://the-guild.dev/graphql/codegen). Using this utility skips all file system traversal and uses the
+ * documents defined in the persisted documents file.
+ *
+ * For more information see the [Persisted documents](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client#persisted-documents) documentation.
+ *
+ * @example
+ * ```ts
+ * import {
+ *   fromGraphQLCodegenPersistedDocuments
+ * } from '@apollo/generate-persisted-query-manifest';
+ *
+ * const config = {
+ *   documents: fromGraphQLCodegenPersistedDocuments('./path/to/persisted-documents.json')
+ * };
+ * ```
+ *
+ * @since 1.2.0
+ */
+export function fromGraphQLCodegenPersistedDocuments(
```

Comment:
Should this functionality also be available on the CLI as an option, or should we generally rely more on the config for this kind of change?

Reply:
Hmm, this seems fine I think.